web/src/app/users/UserNotificationRuleList.js

```diff
@@ -35,37 +34,18 @@ const query = gql`
   }
 `
 
-const styles = (theme) => {
+const useStyles = makeStyles((theme) => {
   const { cardHeader } = globalStyles(theme)
-
   return {
     cardHeader,
   }
-}
-
-@withStyles(styles)
-export default class UserNotificationRuleList extends React.PureComponent {
-  static propTypes = {
-    userID: p.string.isRequired,
-    readOnly: p.bool,
-  }
-
-  state = {
-    delete: null,
-  }
+})
 
-  render() {
-    return (
-      <Query
-        query={query}
-        variables={{ id: this.props.userID }}
-        render={({ data }) => this.renderList(data.user.notificationRules)}
-      />
-    )
-  }
+export default function UserNotificationRuleList({ userID, readOnly }) {
+  const classes = useStyles()
+  const [deleteId, setDeleteId] = useState(null)
```

Comment:
let's capitalize "id" throughout to keep with convention
```suggestion
  const [deleteID, setDeleteID] = useState(null)
```